stdlib/public/core/Runtime.swift.gyb

```diff
@@ -226,7 +228,7 @@ public func _swift_stdlib_atomicLoadInt(
 #elseif arch(x86_64) || arch(arm64) || arch(powerpc64) || arch(powerpc64le) || arch(s390x)
   return Int(Int64(bitPattern:
     _swift_stdlib_atomicLoadUInt64(
-      object: UnsafeMutablePointer(target))))
+        object: unsafeBitCast(target, to: UnsafeMutablePointer<UInt64>.self))))
```

Comment:
In all of these cases, emacs thinks the line after the return should be 4-space indented and the rest by 2-spaces:

return Int(Int64(bitPattern:
[ ][ ][ ][ ]_swift_stdlib(...
[ ][ ][ ][ ][ ][ ]object:

I'd prefer not to hand edit these things, but I'm not sure the emacs style is acceptable.